spec/system/admin_kudos_delete_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe '#delete?', type: :system do
+  let!(:admin) { create :admin }
+  let!(:employee) { create :employee }
+  let!(:kudo) { create :kudo, giver: employee }
```

Comment:
Suggestion, if the kudo factory is correctly written, you don't have to create manually employee, and the set it as a giver, it will be automatically created by factorybot

Reply:
Right, I made some minor changes here and everything works fine